scan_to_inventory/static/www/controllers/c_load.js

```diff
@@ -0,0 +1,55 @@
+angular.module('scan_to_inventory').controller(
+        'LoadCtrl', [
+        '$scope', '$rootScope', 'jsonRpc', '$state', 'ProductProductModel', 'StockLocationModel', 'StockInventoryModel', '$translate',
+        function ($scope, $rootScope, jsonRpc, $state, ProductProductModel, StockLocationModel, StockInventoryModel, $translate) {
+
+    $scope.data = {};
+
+    $scope.$on(
+            '$stateChangeSuccess',
+            function(event, toState, toParams, fromState, fromParams){
+        if ($state.current.name === 'load') {
+            $scope.data.product_ok = false;
+            $scope.data.product_ko = false;
```

Comment:
While I was reading your code I didn't get why you had product_ok and product_ko at the same time. You can may be choose better names likes  or may be use only one variable: 
`product_loaded_correctly = true | false | undefined`
(or you can even add the value: `product_qty = number | false | undefined`)